packages/atomic/src/pages/examples/tabs.html

```diff
@@ -253,12 +257,12 @@
                 label="Least Recent"
                 expression="date ascending"
               ></atomic-sort-expression>
-              <atomic-sort-expression label="relevance" expression="relevancy"></atomic-sort-expression>
             </atomic-sort-dropdown>
             <atomic-did-you-mean></atomic-did-you-mean>
             <atomic-notifications></atomic-notifications>
           </atomic-layout-section>
           <atomic-layout-section section="results">
+            <atomic-generated-answer tabs-included='["all","parts"]'></atomic-generated-answer>
```

Comment:
```suggestion
            <atomic-generated-answer tabs-included="['all','parts']"></atomic-generated-answer>
```

Reply:
Would be invalid JSON, all the `@ArrayProp` props need to use that format.